### PR TITLE
ALFREDAPI-480: Render lists correctly

### PR DESCRIPTION
**Pull request by WimCrols**
and other minor orthographic fixes.

### CODE REVIEW

File: user/user-guide.md [hunk 1]
```diff
@@ -532,6 +534,7 @@ Expression based sorting is not implemented.
 `Optional`
 
 Option to request specific consistency. Options are:
+
```
Comment (WimCrols) [inline user/user-guide.md:537]: Leading newlines fix list (and code block) rendering

File: user/user-guide.md [hunk 2]
```diff
@@ -697,21 +705,20 @@ The search REST call returns a JSON object of the following form:
...
 }
-
 ```
 
 ### Additional notes
 
 #### Date format
 
-Dates and times must be specified int the ISO-8601 format
+Dates and times must be specified in [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html) format.
```
Comment (RVanhuysseXenit) [inline user/user-guide.md:714]: (non-blocking) Maybe more useful to link to the corresponding rfc https://tools.ietf.org/html/rfc3339

File: user/user-guide.md (hunk 1 above)
Comment (RVanhuysseXenit) [reply user/user-guide.md:540]: glad it was as simple as that

**Review by RVanhuysseXenit — APPROVED** [review]

File: user/user-guide.md (hunk 2 above)
Comment (WimCrols) [reply user/user-guide.md:714]: Though technically more correct, reading the IETF link would take 10min, and reading this link takes 10s :)

**Pull request merged**